src/node/coinstats.h

```diff
@@ -24,7 +24,10 @@ struct CCoinsStats
     uint64_t nDiskSize;
     CAmount nTotalAmount;
 
-    CCoinsStats() : nHeight(0), nTransactions(0), nTransactionOutputs(0), nBogoSize(0), nDiskSize(0), nTotalAmount(0) {}
+    // The number of coins contained.
+    uint64_t coins_count;
+
+    CCoinsStats() : nHeight(0), nTransactions(0), nTransactionOutputs(0), nBogoSize(0), nDiskSize(0), nTotalAmount(0), coins_count(0) {}
```

Comment:
I'd prefer to use C++11 member initialization

```cpp
uint64_t coins_count{0};
```

Feel free to keep the others (or switch them over as well)

Reply:
Changed them all.